integration/test/performancetest/performance_test.go

```diff
@@ -1,59 +1,158 @@
-// go:build linux && integration
+//go:build linux && integration
 // +build linux,integration
 
 package performancetest
 
-import(
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"sync"
 	"testing"
 	"time"
-	"log"
-	"context"
+
 	"github.com/aws/amazon-cloudwatch-agent/integration/test"
 )
 
 const (
-	configPath = "resources/config.json"
 	configOutputPath = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
-	agentRuntimeMinutes = 5 //20min desired but 5mins for testing purposes 
+	agentRuntimeMinutes = 5 //20 mins desired but 5 mins for testing purposes
 	DynamoDBDataBase = "CWAPerformanceMetrics"
-
 )
 
 func TestPerformance(t *testing.T) {
+	configFiles := []string {
+		"./resources/config10Logs.json",
+		"./resources/config100Logs.json",
+		"./resources/config1000Logs.json"
+	}
+
+	tpsVals := []int {
+		10,
+		100,
+		1000
+	}
+	
 	agentContext := context.TODO()
 	instanceId := test.GetInstanceId()
 	log.Printf("Instance ID used for performance metrics : %s\n", instanceId)
 
-	test.CopyFile(configPath, configOutputPath)
+	//data base
+	dynamoDB := InitializeTransmitterAPI(DynamoDBDataBase) //add cwa version here
+	if dynamoDB == nil {
+		t.Fatalf("Error: generating dynamo table")
+	}
+
+	//run tests
+	for _, configPath := range configFiles {
+		for _, tps := range tpsVals {
+			t.Run(fmt.Sprintf("config file location: %s", configPath), func(t *testing.T) {
```

Comment:
I think there's going to be a lot of overlap in these test names. We should include some information on the TPS as well. Maybe something like 
```suggestion
			t.Run(fmt.Sprintf("config file location: %s, TPS: %d", configPath, tps), func(t *testing.T) {
```